### DISCUSSION TITLE
Helm Chart - v1.7.2

### DISCUSSION THREAD

**Discussion by tomkerkhove** [Azure/api-management-self-hosted-gateway#233] [Announcements]
[![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/azure-api-management)](https://artifacthub.io/packages/helm/azure-api-management/azure-api-management-gateway/1.7.2)

*Chart: v1.7.2 | App: v2.3.2*

### Features

- Use container image v2.3.2 ([release notes](https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/Container-v2.3.2))

### Fixes / Changes

None.

### Breaking Changes

None.

### Removal

None.

## New Contributors

None.

**Full Changelog**: https://github.com/Azure/api-management-self-hosted-gateway/compare/v1.7.1...v1.7.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/v1.7.2'>Helm Chart - v1.7.2</a>.</em>

**Comment by oskrabanek** [comment]
The package is not in the artifact hub.
![image](https://github.com/Azure/api-management-self-hosted-gateway/assets/6212595/de0f0e73-9686-4f4f-a0b0-c2376dc6d925)

**Reply by tomkerkhove** [reply]
Thanks for reporting: https://github.com/Azure/api-management-self-hosted-gateway/issues/236
